src/Files.App/Dialogs/CredentialDialog.xaml

```diff
@@ -22,31 +22,42 @@
 		</ResourceDictionary>
 	</ContentDialog.Resources>
 
-	<Grid RowSpacing="25">
+	<Grid RowSpacing="24">
```

Comment:
Thanks!

Reply:
There're many:) 25 changed to 24, 50 changed to 50.